core/src/main/java/org/infinispan/factories/InterceptorChainFactory.java

```diff
@@ -12,28 +12,9 @@
 import org.infinispan.configuration.cache.InterceptorConfiguration;
 import org.infinispan.configuration.cache.StoreConfiguration;
```

Comment:
is this factory still creating ```InterceptorChain```?
If not, it can be removed from the annotation.